Zotero/Scenes/AppCoordinator.swift

```diff
@@ -368,8 +382,12 @@ final class AppCoordinator: NSObject {
         imageView.frame = window.bounds
 
         // Create a temporary `UIWindow` which will be shown above current window until it successfully presents the new view controller.
-        let tmpWindow = UIWindow(frame: window.frame)
-        tmpWindow.windowScene = window.windowScene
+        let tmpWindow: UIWindow
+        if let windowScene = window.windowScene {
+            tmpWindow = UIWindow(windowScene: windowScene)
+        } else {
+            tmpWindow = UIWindow(frame: window.frame)
+        }
```

Comment:
You could simplify this into one liner:
`let tmpWindow = window.windowScene.flatMap(UIWindow.init) ?? UIWindow(frame: window.frame)`

Reply:
Check https://github.com/zotero/zotero-ios/pull/821/commits/93c1c0b001ce45d02cdd8da350908d2c0bff8ab1